style.css

```diff
@@ -250,7 +300,56 @@ body {
   margin: 0 16px 0 0;
 }
 
-@media screen and (min-width: 375px) {
+
+@media screen and (max-width: 768px) {
```

Comment:
Since you are using @media screen and (min-width: 375px) for the mobile version, I believe it would be beneficial to follow the same concept in the desktop version as well. For example, you can use (min-width: 768px) instead of (max-width: 768px). By doing so, you can achieve a more readable and cleaner version of the code.

The reason for this is that when we already use (min-width: 375px) for the mobile version, it makes sense to be consistent and use (min-width: 768px) for the desktop version. This ensures a more coherent and maintainable approach in your media queries 👍 👍 .